packages/wrangler/src/pages/build.ts

```diff
@@ -45,6 +45,10 @@ export function Options(yargs: CommonYargsArgv) {
 				type: "string",
 				description: "The location for the output config file",
 			},
+			"build-metadata-path": {
```

Comment:
what is this for?

Reply:
This is for integration with CI—perhaps it shouldn't be in this PR